internal/services/topic.go

```diff
@@ -0,0 +1,166 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package services defines some canary related services
+package services
+
+import (
+	"log"
+	"math"
+	"os"
+	"strconv"
+
+	"github.com/Shopify/sarama"
+	"github.com/strimzi/strimzi-canary/internal/config"
+)
+
```

Comment:
In general in this file I feel that a little more documentation would go along way. There are a lot of ints flying around and I don't know whether they're numbers of brokers, broker ids, numbers of replicas or replication factors.

Reply:
Yes, we need it, let me add it in a different PR. Opening an issue for tracking for now.